addons/website_sale/static/src/js/product_list/product_list.js

```diff
@@ -12,4 +12,8 @@ patch(ProductList.prototype, {
             this.optionalProductsTitle = _t("Available options");
         }
     },
+
+    get totalMessage() {
+        return _t("Total: %s", this.getFormattedTotal());
+    },
```

Comment:
IMO, if you keep this implementation, this should call super if env is not frontend

Reply:
Yep, I can add an extra check :) I didn't bother initially since I guess there's no harm in always overriding the message but I guess it's a bit cleaner